src/py42/_internal/clients/pds.py

```diff
@@ -0,0 +1,24 @@
+from py42.clients import BaseClient
+
+
+class PreservationDataServiceClient(BaseClient):
+    def find_file_versions(self, file_md5, file_sha256, device_id, paths):
+        """Fetch file version details.
+
+        Args:
+            file_md5 (str): MD5 encoded hash of the file.
+            file_sha256 (str): SHA256 encoded hash of the file.
+            device_id (str): Device id.
```

Comment:
Could use more info for `device_id`, like `Id of the device where the files are from.` or something

Reply:
Done.